Source/Engine/Platform/Linux/LinuxWindow.cpp

```diff
@@ -594,6 +594,12 @@ void LinuxWindow::OnButtonPress(void* event)
 	case Button3:
 		mouseButton = MouseButton::Right;
 		break;
+	case 8:
```

Comment:
I don't know why this, but Linux API don't have constants to extended buttons, only Button1, 2, 3, 4, 5 but don't have for 8 and 9.

Reply:
Yeah typical of `X11` API :D thx for this improvement